README.md

```diff
@@ -45,7 +45,7 @@ on its own, run `yarn workspace frontend start` or `yarn workspace backend start
 - **Heewon Ahn** - Developer
 - **Kinjal Jasani** - Developer
 - **Mena Attia** - Developer
-- **Sophia Pham** - Developer
+- **Thuy (Sophia) Pham** - Developer
```

Comment:
Make sure to rebase your branch with `main`! For example, this README change should not be here